getting-started/troubleshooting/error-handling.rst

```diff
@@ -228,18 +227,49 @@ a mistake was made entering the permalink:
    Warning that permalink was incorrect, allows user to retry upload DIP
 
 
-.. _common-errors:
+.. _failure-errors:
 
-Other common error behaviours
+Errors that fail transfers
 -----------------------------
 
-#. Verify metadata directory checksums: if the checksums in the metadata
-   directory cannot be verified (i.e. if a file is missing or corrupted) the
-   microservice will fail and the transfer will be moved in the failed
+There are a number of microservice errors that will halt the workflow and cause
+Archivematica to report a 'Failed transfer'. It is important to note that not
+all microservice errors will lead to a failed transfer. Below is a list of
+common errors that will result in the 'Failed transfer' microservice to run and
+the transfer to be moved to the failed directory.
+
+#. Approve Transfer (zipped and unzipped bags): When running a zipped or
```

Comment:
This is a very minor thing, but an unordered list might be better than an ordered list here? I totally leave this up to you though.

Reply:
valid point. It saves future edits from making sure they remain in order.